### PR TITLE
Implement option --quick-test and add GPU functionality.

### PR DESCRIPTION
**Pull request by AGrigis**

### CODE REVIEW

File: environment.yml
Comment (agramfort) [inline environment.yml]: please revert this as we will need the latest version on the server to deploy the challenge in the coming days. This ramp-kit must work with the latest ramp-workflow on the pypi

**Review by agramfort — APPROVED** [review]
green !

@frcaud I let you test before merging ok?

**Comment by frcaud** [comment]
LGTM provided that we set 'cnt==2' in ```get_cv``` and replace 'np.nan' by '0' in 'challenge_metric' and 'ext_mae_age'

**Comment by frcaud** [comment]
Thx @AGrigis !

**Pull request merged**
